noir-projects/noir-contracts/contracts/avm_test_contract/src/main.nr

```diff
@@ -17,6 +17,16 @@ impl Deserialize<2> for Note {
     }
 }
 
+impl Packable<2> for Note {
+    fn pack(self) -> [Field; 2] {
+        [self.a, self.b]
+    }
+
+    fn unpack(fields: [Field; 2]) -> Self {
+        Note { a: fields[0], b: fields[1] }
+    }
+}
+
```

Comment:
Serialize is fine here

Reply:
Same as above. Packable required for public storage